standard/stream_links.lua

```diff
@@ -51,4 +55,54 @@ function StreamLinks.readCountdownStreams(args)
 	return stream
 end
 
+--[[
+Resolves the value of a stream given the platform
+]]
+function StreamLinks.resolve(platformName, streamValue)
+	local streamLink = mw.ext.StreamPage.resolve_stream(platformName, streamValue)
+
+	return string.gsub(streamLink, 'Special:Stream/' .. platformName, '')
+end
+
+--[[
+Extracts the streaming platform args from an argument table or a nested stream table inside the arguments table.
+Uses variable fallbacks and resolves stream redirects.
+]]
+function StreamLinks.processStreams(args)
```

Comment:
better parameter name

Reply:
hmmm
- it might be used outside of matches too, which is why i didn't use `match`
- the table passed here most likely does not only contain arguments for the stream table but also other stuff (for match it certainly does), which is why i didn't use `streamArgs`

if you have a better name for `args` i am all ears

Reply:
You could use `match2Args`

Reply:
i mean i am fine with any name for the args, but since that function is (theoretically) usable outside of match/match2 it might be a bit misleading

Reply:
`streams`?

Reply:
Let's go with `forwardedInputArgs`

Reply:
done